docs/en/setup/service-agent/java-agent/Application-toolkit-trace.md

```diff
@@ -53,3 +53,14 @@ public User methodYouWantToTrace(String param1, String param2) {
 }
 ```
 
+* Use `TraceContext.getCorrelation()` API to put custom data in tracing context. 
+```java
+Optional<String> previous = TraceContext.getCorrelation("customKey", "customValue");
```

Comment:
I think this is a typo of the documentation? get with 2 parameters?

Reply:
My mistake. Write in the wrong place.